network/drivers/bridge.md

```diff
@@ -91,14 +92,47 @@ network.**
     compose file can define the shared variables.
 
   - You can use swarm services instead of standalone containers, and take
-    advantage of shared [secrets](../engine/swarm/secrets.md) and
-    [configs](../engine/swarm/configs.md).
+    advantage of shared [secrets](../../engine/swarm/secrets.md) and
+    [configs](../../engine/swarm/configs.md).
 
 Containers connected to the same user-defined bridge network effectively expose all ports
 to each other. For a port to be accessible to containers or non-Docker hosts on
 different networks, that port must be _published_ using the `-p` or `--publish`
 flag.
 
+## Options
+
+The following table describes the driver-specific options that you can pass to
+`--option` when creating a custom network using the `bridge` driver.
+
+| Option                                           | Default        | Description                                                 |
+| ------------------------------------------------ | -------------- | ----------------------------------------------------------- |
+| `com.docker.network.bridge.name`                 |                | Interface name to use when creating the Linux bridge.       |
```

Comment:
I think we should mention that the kernel doesn't allow an interface name to be more than 15 chars long.

Reply:
As for the restrictions on netdev naming: https://unix.stackexchange.com/a/677481